src/api/app/views/webui/monitor/old.html.erb

```diff
@@ -1,5 +1,7 @@
 <% @pagetitle = 'Build Status Monitor' %>
-<% @crumb_list = %w(Monitor) %>
+<% @crumb_list = [] %>
+<% @crumb_list << link_to('Monitor', :controller => 'monitor', :action => :index) %>
+<% @crumb_list << %w(Detailed) %>
```

Comment:
That would result in [ "Monitor", ["Detailed"]]. Probably not intended.

This could also be shortened to ```<% @crumb_list = [link_to("Monitor", monitor_path), "Detailed"] %>```